app/models/game.rb

```diff
@@ -6,11 +6,11 @@ class Game < ApplicationRecord
 
   validates :name, presence: true
 
-  has_many :categories
+  has_many :categories, dependent: :destroy
   has_many :runs, through: :categories
   has_many :users, -> { distinct }, through: :runs
   has_many :runners, -> { distinct }, through: :runs, class_name: 'User'
-  has_many :aliases, class_name: 'GameAlias'
+  has_many :aliases, class_name: 'GameAlias', dependent: :destroy
```

Comment:
Hmm, I actually can't think of any current or future use case for destroying a game other than the existing [merge flow][1], and in that case any aliases and categories should have already been ported over to the new game. It sounds a bit weird in theory, but realistically I'd consider the destroying of a game that still has aliases or categories to be a bug, so maybe these should be `:restrict_with_exception`.

[1]: https://github.com/glacials/splits-io/blob/master/app/models/game.rb#L74-L87

Reply:
This broke the tests, and since it was only supposed to be a rubocop fix, I'll leave this for another time.